app/screens/send-bitcoin-screen/send-bitcoin-screen.tsx

```diff
@@ -407,9 +485,36 @@ export const SendBitcoinScreenJSX: ScreenType = ({
             currency={secondaryAmount.currency}
             style={styles.subCurrencyText}
           />
+          {minSendable !== "" && (
+            <View style={styles.errorContainer}>
+              <Text>Min: {minSendable} sats - Max: {maxSendable} sats</Text>
+            </View>
+          )}
         </View>
 
         <View style={{ marginTop: 18 }}>
+          {paymentType === "lnurl" && (<GaloyInput
+            placeholder={translate("common.domain")}
+            style={styles.smallText}
+            leftIcon={
+              <View style={styles.row}>
+                <Text style={styles.smallText}>{translate("common.domain")}</Text>
+                <Icon
+                  name="ios-at"
+                  size={24}
+                  color={color.primary}
+                  style={styles.icon}
+                />
+              </View>
+            }
+            onChangeText={setDestination}
+            rightIcon={destinationInputRightIcon()}
+            value={lnurlDomain}
+            editable={interactive}
+            selectTextOnFocus
+            autoCompleteType="username"
+            autoCapitalize="none"
+          />)}          
```

Comment:
Let's not use an input element for this.

Reply:
what should I use?

Reply:
I think a View/Text here should be fine. Maybe under Note input.